src/resources/postcss/tickets-admin.pcss

```
@@ -613,3 +613,47 @@ p.description {
 #ticket_form .original_stock_cap span {
 	margin-right: 0;
 }
+
+/* -- Event Tickets tweaks -- */
+.tribe-events-admin-tickets {
+	.tribe-events-admin-quick-nav {
+		@media screen and (min-width: 750px) {
+			width: 750px;
+		}
+	}
```

I _think_ our pcss rules want these spaced like so:

```suggestion
.tribe-events-admin-tickets {

	.tribe-events-admin-quick-nav {
	
		@media screen and (min-width: 750px) {
			width: 750px;
		}
	}
```